snapper/Btrfs.cc

```diff
@@ -314,7 +318,7 @@ namespace snapper
 #ifdef ENABLE_ROLLBACK
 
     void
-    Btrfs::createSnapshotOfDefault(unsigned int num, bool read_only) const
+    Btrfs::createSnapshotOfDefault(unsigned int num, bool read_only, bool quota) const
```

Comment:
If those boolean parameters grow any more, maybe OR'ed flags might be a good idea. Not right now with two of them, but if there will be a third, I'd strongly recomend this.